packages/nuxt/src/runtime/plugins/sentry.client.ts

```diff
@@ -34,18 +35,35 @@ export default defineNuxtPlugin({
   name: 'sentry-client-integrations',
   dependsOn: ['sentry-client-config'],
   async setup(nuxtApp) {
+    const sentryClient = getClient();
+    const clientOptions = sentryClient && sentryClient.getOptions();
+
     // This evaluates to true unless __SENTRY_TRACING__ is text-replaced with "false", in which case everything inside
     // will get tree-shaken away
     if (typeof __SENTRY_TRACING__ === 'undefined' || __SENTRY_TRACING__) {
-      const sentryClient = getClient();
-
       if (sentryClient && '$router' in nuxtApp) {
         sentryClient.addIntegration(
           browserTracingIntegration({ router: nuxtApp.$router as VueRouter, routeLabel: 'path' }),
         );
       }
     }
 
+    if (clientOptions && 'trackPinia' in clientOptions && clientOptions.trackPinia) {
+      if ('$pinia' in nuxtApp) {
+        (nuxtApp.$pinia as { use: (plugin: unknown) => void }).use(
+          createSentryPiniaPlugin(clientOptions.trackPinia === true ? undefined : clientOptions.trackPinia),
```

Comment:
l: At this point `trackPinia` is already true, so I think we can skip the ternary

Reply:
It can be `true` or an object. If it is true, the default options should be used (that's why undefined is passed). But I can add a comment to clarify this.

Reply:
ah yeah I missed that, no need for the comment 👍